packages/backend-modules/auth/lib/AccessToken.js

```diff
@@ -138,13 +138,20 @@ const resolve = async (token, { pgdb }) => {
   const [userId, scope, expiresAt, hmac] = base64u.decode(token).split('/')
   if (userId && scope && expiresAt && hmac) {
     const payload = getPayload({ userId, scope, expiresAt })
-    const key = await pgdb.public.users.findOneFieldOnly(
-      { id: userId },
-      'accessKey',
-    )
+    const userRecord = await pgdb.public.users.findOne({ id: userId })
+    if (userRecord.revokedAt) {
```

Comment:
`revokedAt` does not exist in user rows.

(This will always default to `true`.)